src/main.ts

```diff
@@ -0,0 +1,50 @@
+import { match, MatchFunction, MatchResult } from "path-to-regexp";
+import { handleImportMap } from "./handleImportMap";
+import { notFoundResponse } from "./notFound";
+
+addEventListener("fetch", (evt: FetchEvent) => {
+  evt.respondWith(handleRequest(evt.request));
+});
+
+const routeHandlers: RouteHandlers = {
+  "/:orgKey/:importMapName.importmap": handleImportMap,
+};
+
+const routeMatchers: RouteMatchers = Object.entries(routeHandlers).map(
+  ([path, handler]) => [match(path), handler]
+);
+
+async function handleRequest(request: Request) {
+  const requestUrl = new URL(request.url);
+
+  let routeHandler: RouteHandler | undefined,
+    matchResult: MatchResult | false = false;
```

Comment:
Could this also be a `let`?

Reply:
It is. You can declare multiple variables at once by doing `let var1, var2`. Both var1 and var2 in that example are `let` variables. In the case of the code here, both `routeHandler` and `matchResult` are `let` variables, declared in the same statement and separated by a comma on line 20